cmd/podman/main.go

```diff
@@ -120,7 +120,7 @@ func main() {
 			os.Exit(1)
 		}
 		args := c.Args()
-		if args.Present() {
+		if args.Present() && rootless.IsRootless() {
```

Comment:
@giuseppe Is this safe (is rootless.IsRootless() set appropriately at this point?)

Reply:
hes the one who advised to do this :)  but lets make sure